tests/suites/test_suite_ssl.function

```diff
@@ -5,6 +5,35 @@
 #include <mbedtls/entropy.h>
 #include <mbedtls/certs.h>
 #include <mbedtls/timing.h>
+#include <mbedtls/debug.h>
+
+
+typedef struct log_pattern
+{
+    const char *pattern;
+    size_t counter;
+} log_pattern;
+
+/* This function can be passed to mbedtls to receive output logs from it. In
+ * this case, it will count the instances of a log_pattern in the received
+ * logged messages.
+ */
+static void log_analyzer( void *ctx, int level,
+                          const char *file, int line,
+                          const char *str )
+{
+    (void) level;
+    (void) line;
+    (void) file;
+
+    log_pattern *p = (log_pattern*) ctx;
+    if( NULL != p &&
```

Comment:
This formatting is a bit strange (since you're well below the line limit), but it's a minor.